lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -2761,24 +2761,29 @@ galaxy:
   # only if you have setup a Celery worker for Galaxy and you have
   # configured the `celery_conf` option below. Specifically, you need to
   # set the `result_backend` option in the `celery_conf` option to a
-  # valid Celery result backend URL. For details, see
+  # valid Celery result backend URL. By default, Galaxy uses an SQLite
+  # database at '<data_dir>/results.sqlite' for storing task results.
+  # Please use a more robust backend (e.g. Redis) for production setups.
```

Comment:
I think what you added here would be more useful directly in the documentation of `celery_conf` below?

Reply:
That makes sense, thank you!